core/src/parameters.rs

```diff
@@ -31,8 +31,13 @@ pub mod consensus_internal {
     pub const REWARD_EPOCH_COUNT: u64 = 12;
     pub const ANTICONE_PENALTY_UPPER_EPOCH_COUNT: u64 = 10;
     pub const ANTICONE_PENALTY_RATIO: u64 = 100;
-    /// 900 Conflux tokens
-    pub const BASE_MINING_REWARD: u64 = 900;
+    /// 17 Conflux tokens will give 1B tokens for the first year.
+    /// Consider we stick with the schedule of halving every 3.5 years.
+    /// That means we need to reduce the reward with a ratio of 0.951695 every
+    /// three months Therefore to release 1B for the first year, the first
+    /// three month we need to release 268861235 tokens. 268861235 / (365 *
+    /// 24 * 60 * 60 / 4) / 2 (two blocks per second) = 17.05
+    pub const BASE_MINING_REWARD: u64 = 17;
     /// The unit of one Conflux token: 10 ** 18
     pub const CONFLUX_TOKEN: u64 = 1_000_000_000_000_000_000;
```

Comment:
This comment isn't for this PR.

I suggest change the naming. And BTW when I checked usage of CONFLUX_TOKEN, I find RENTAL_PRICE_PER_STORAGE_KEY not accurate.